pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[project]
+name = "exhibitors"
+dynamic = ["version"]
+description = "Eventyay-tickets plugin to add and manage exhibitors"
+readme = "README.rst"
+requires-python = ">=3.9"
```

Comment:
Do we need to support Python as old as v3.9?

Reply:
I just added it as a basic requirement can be changed to a later version

Reply:
No need for 3.9. Let's stick to newer versions.

Reply:
I have updated it to **3.11**